Unified/closor.py

```diff
@@ -551,30 +551,30 @@ def close(self):
-                            ddm_instructions = CI.campaigns[campaign]['DDMcopies']
-                            if type(ddm_instructions) == int:
-                                n_copies = CI.campaigns[campaign]['DDMcopies']
-                            elif type(ddm_instructions) == dict:
-                                ## a more fancy configuration
-                                for ddmtier,indication in ddm_instructions.items():
-                                    if ddmtier==tier or ddmtier in ['*','all']:
-                                        ## this is for us
-                                        if 'N' in indication:
-                                            n_copies = indication['N']
-                                        if 'host' in indication:
-                                            destinations = indication['host']
+                        #destinations=[]
+                        #if to_DDM and campaign and campaign in CI.campaigns and 'DDMcopies' in CI.campaigns[campaign]:
+                        #    ddm_instructions = CI.campaigns[campaign]['DDMcopies']
+                        #    if type(ddm_instructions) == int:
+                        #        n_copies = CI.campaigns[campaign]['DDMcopies']
+                        #    elif type(ddm_instructions) == dict:
+                        #        ## a more fancy configuration
+                        #        for ddmtier,indication in ddm_instructions.items():
+                        #            if ddmtier==tier or ddmtier in ['*','all']:
+                        #                ## this is for us
+                        #                if 'N' in indication:
+                        #                    n_copies = indication['N']
+                        #                if 'host' in indication:
+                        #                    destinations = indication['host']
                                             
-                        destination_spec = ""
-                        if destinations:
-                            destination_spec = "--destination="+",".join( destinations )
-                        group_spec = "" ## not used yet 
+                        #destination_spec = ""
+                        #if destinations:
+                        #    destination_spec = "--destination="+",".join( destinations )
+                        #group_spec = "" ## not used yet 
                         ### should make this a campaign configuration
                         ## inject to DDM when necessary
                         if to_DDM:
                             print "Sending",out," to DDM"
-                            status = pass_to_dynamo( [out], N = n_copies, sites=destinations if destinations else None, group = group_spec if group_spec else None)
+                            status = pass_to_dynamo( [out], N = n_copies, sites=destinations if destinations else None, None)
```

Comment:
`destinations` is not initialized inside this if-else block. Either you initialize it as done before `destinations=[]`, or you update this `pass_to_dynamo` line to `sites=None`

Reply:
I think there is more here that we are missing.` [L531-L551](https://github.com/CMSCompOps/WmAgentScripts/blob/master/Unified/closor.py#L531-L551)` is used now but I am not able to see the transfers that should happen for these if `to_DDM` is `true`.